### DISCUSSION TITLE
Usercard for ID

### DISCUSSION THREAD

**Discussion by Nerixyz** [Chatterino/chatterino2#4724] [Ideas] [answered]
Currently, you can only open user cards for a specific username. Sometimes you have an ID and want to get info about that user.

I propose the `/usercardid <id> [channel]` command. It works like the `/usercard <name> [channel]`, except that you give it an ID. I created a basic implementation in https://github.com/Nerixyz/chatterino2/commit/a75f92747cf1151623dc6d5a5d8268c9fd244bf2.

**Comment by 8thony** [comment, marked as answer]
- Implemented since #4934 

Usage: `/usercard id:<id> [channel]`
